File: src/main/java/com/recruit/entity/Company.java
package com.recruit.entity;

import lombok.Data;
import org.springframework.stereotype.Component;

import java.util.List;

@Component
@Data
public class Company {
    private Integer id;
    private String username;
    private String password;
    private String name;
    private String address;
    private String email;
    private String phone;
    private String introdu;

    public Company() {
    }

    public Company(String username, String password) {
        this.username = username;
        this.password = password;
    }
}

File: src/main/webapp/company/js/company.js
$(function () {
    $.ajax({
        url: "findCompany",
        type: "POST",
        success: function (data) {
            console.log(data)
            $("#id").val(data.id)
            $("#username").val(data.username)
            $("#password").val(data.password)
            $("#name").val(data.name)
            $("#address").val(data.address)
            $("#email").val(data.email)
            $("#phone").val(data.phone)
            $("#introdu").val(data.introdu)
        }
    })

    $("#update").click(function () {
        $.ajax({
            url: "updateCompany",
            type: "POST",
            data: $("#publish").serialize(),
            success: function (data) {
                if (data > 0) {
                    alert("修改成功！")
                    parent.location.reload(); //刷新父亲对象（用于框架）
                }
            }
        })
    })

})